### PR TITLE
Add FAQ page with info on line endings

### PR DESCRIPTION
**Pull request by WasabiFan**
I _really_ hate reStructured Text. 😑

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+-----
+
+This may occur if your file includes Windows-style line endings, which are often
+inserted by editors on Windows. To resolve this issue, open an SSH session and
+run the following command, replacing ``<file>`` with the name of the Python file
+you're using:
+
+.. code::
+
+    sed -i 's/\r//g' <file>
```
Comment (ddemidov) [inline docs/faq.rst]: There used to be a dos2unix utility, not sure if it's available by default on the ev3 though.

Comment (WasabiFan) [inline docs/faq.rst]: It still exists but is not installed by default, so I decided to go with the built-in option.

Comment (ddemidov) [inline docs/faq.rst]: :+1:

File: docs/faq.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
+Frequently-Asked Questions
+==========================
+
+When I try to run my Python file from the shell or Brickman,
+it throws errors or exits immediately
+-----
```
Comment (WasabiFan) [inline docs/faq.rst]: You can tell I was distracted while writing this :laughing: Three times I wrapped this line, and three times I realized it was broken and fixed it.

**Comment by WasabiFan** [comment]
I couldn't think of a better title for the entry, but I'm not particularly happy with this one. If you have a better idea I'd happily change it.

**Comment by ddemidov** [comment]
> My script works when launched as `python3 script.py` but exits immediately when launched from brickman or as `./script.py`?

Seems as long as the original though.

**Comment by WasabiFan** [comment]
I like that; it explains the issue a bit more explicitly. Look good with that swap?

**Comment by ddemidov** [comment]
http://ev3dev-lang.readthedocs.io/projects/python-ev3dev/en/latest/faq.html

The page title misses an 'F' for me

**Comment by ddemidov** [comment]
Not sure how did I miss it during review :open_mouth:

**Comment by WasabiFan** [comment]
https://github.com/rhempel/ev3dev-lang-python/commit/251b3a71444517d4af3cc22d5834fe7bb7206fec#commitcomment-19706795 Working on a fix

**Comment by WasabiFan** [comment]
It might take me a few hours though 😆

**Comment by ddemidov** [comment]
Take your time, we want it perfect this time

**Comment by WasabiFan** [comment]
I'm trying to link to the FAQ page from the README. How should I do that? Link to the (currently nonexistent) page on `stable`, or link to `latest`? Or maybe to the RST file on GitHub?

**Comment by ddemidov** [comment]
I am starting to think we should point the README docs badge to the latest branch, or maybe provide two badges.

The README ends up as a frontpage for readthedocs, so I don't think we should link to the `latest` docs from the `stable` README. On the other hand, the README that is shown on github is the `latest` one (`develop`). Linking to the RST file github does not seem right as well. May be link to the latest version for now, and switch it to the stable when we are ready to tag 0.8.1? We could add an issue for this and link it to 0.8.1 milestone so that we don't forget.

**Comment by WasabiFan** [comment]
> I am starting to think we should point the README docs badge to the latest branch, or maybe provide two badges.

Honestly, the idea set-up in my eyes would be to have GitHub show the `master` branch by default and everything would be fine, in the sense that having links to `stable` from the development branch is much less of an issue than the inverse because it doesn't affect the average user. However, the argument I've heard against this (which I believe is a good one) is that we want people who contribute code to do it for the `develop` branch. It's a difficult thing to manage over all.

> May be link to the latest version for now, and switch it to the stable when we are ready to tag 0.8.1? We could add an issue for this and link it to 0.8.1 milestone so that we don't forget.

I'm thinking that that is the best option for now. I'll do it.

**Comment by ddemidov** [comment]
> However, the argument I've heard against this (which I believe is a good one) is that we want people who contribute code to do it for the develop branch. 

Now that github allows to switch target branches without reopening the PR, this should be less of an issue.

**Pull request merged**